### DISCUSSION TITLE
Github Sponsoring

### DISCUSSION THREAD

**Discussion by FahdSeddik** [community/community#20446] [Other Feature Feedback, Questions, & Ideas] [answered]
How can I enable sponsoring on my profile?

**Comment by andrea-acampora** [comment, marked as answer]
You can find all informations [here](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/displaying-a-sponsor-button-in-your-repository#displaying-a-sponsor-button-in-your-repository)

**Comment by thuongtruong1009** [comment]
Hi @FahdSeddik 
You can read and following step by step at [here](https://github.com/sponsors)
Example in [my repo](https://github.com/thuongtruong1009/bee-music/blob/main/.github/FUNDING.yml)
